rastervision_pytorch_backend/rastervision/pytorch_backend/examples/semantic_segmentation/isprs_potsdam_nochip.py

```diff
@@ -0,0 +1,136 @@
+from rastervision.core.rv_pipeline import (SemanticSegmentationConfig)
+from rastervision.core.data import (
+    SceneConfig, ClassConfig, DatasetConfig, RasterioSourceConfig,
+    SemanticSegmentationLabelSourceConfig,
+    SemanticSegmentationLabelStoreConfig, PolygonVectorOutputConfig)
+from rastervision.pytorch_backend import (PyTorchSemanticSegmentationConfig)
+from rastervision.pytorch_learner import (
+    ExternalModuleConfig, SemanticSegmentationGeoDataConfig,
+    SemanticSegmentationModelConfig, GeoDataWindowMethod, GeoDataWindowConfig,
+    SolverConfig)
+
+TRAIN_IDS = [
+    '2_10', '2_11', '3_10', '3_11', '4_10', '4_11', '4_12', '5_10', '5_11',
+    '5_12', '6_10', '6_11', '6_7', '6_9', '7_10', '7_11', '7_12', '7_7', '7_8',
+    '7_9'
+]
+VAL_IDS = ['2_12', '3_12', '6_12']
+
+CLASS_NAMES = [
+    'Car', 'Building', 'Low Vegetation', 'Tree', 'Impervious', 'Clutter'
+]
+CLASS_COLORS = [
+    '#ffff00', '#0000ff', '#00ffff', '#00ff00', '#ffffff', '#ff0000'
+]
+
+
+def get_config(runner,
+               raw_uri: str,
```

Comment:
I've actually never run the object detection example. I see that there a number of files there. Is there a guide on how to run the example?

Reply:
https://docs.rastervision.io/en/latest/examples.html#object-detection-cowc-potsdam-cars

Reply:
Note about object detection: I changed the `BBoxParams`'s `format` from `'coco'` to `'albumentations'`, because I was getting some strange non-deterministic errors with the former when using `A.HorizontalFlip`.